torch/distributed/rpc/api.py

```diff
@@ -339,6 +341,19 @@ def _validate_rpc_args(backend, store, name, rank, world_size, rpc_backend_optio
             )
 
 
+T = TypeVar("T")
+GenericWithOneTypeVar = Generic[T]
+
+
+class RRefMeta(_RRefImpl.__class__, GenericWithOneTypeVar.__class__):
```

Comment:
This seems a totally different approach for how the type class implemented, why this need to be done in this way? I feel like the previous approach is easier to understand.

Reply:
It's for supporting Python3.5 which has a different meta class implementation in interpreter.

Reply:
PyTorch has dropped support for python 3.5, so we don't need to add it here.